config/manager/manager.yaml

```diff
@@ -11,15 +11,21 @@ metadata:
   namespace: openshift-monitoring
   labels:
     control-plane: controller-manager
+    app: route-monitor-operator
+    component: operator
```

Comment:
Why the `component` label?

Reply:
this is how elasticsearch does stuff, in case we want to remove a whole component at a time.
I did reference it in the README.md for it to be explicit